### ISSUE TITLE
Pub/Sub Emulator Retry Policy

### ISSUE THREAD

**Issue by radvansky-tomas** [firebase/firebase-tools#3738]
I tried to implement failurePolicy / retries using firebase emulator, but it does not seem to work.
Which is probably, because its not being implemented, yet.

So I would like to ask for such feature.

Also I think we should have page in documentation here:
https://firebase.google.com/docs/emulator-suite/connect_and_prototype

Which should clearly list what features are / not supported (which could be handy for other emulators as well)

**Comment by abeisgoat** [comment]
This is unlikely to be implemented in the near future, but the request for better documentation around supported / unsupported features is heard and we're working on it.

**Comment by gregfenton** [comment]
Lack of this support in the emulator is really problematic for developing & testing highly concurrent workflows.  Being forced to do it in a cloud instance rather than locally in the emulator is a burden to developer productivity.

+1 for adding Retry support.

**Comment by ytetsuro** [comment]
duplicated  https://github.com/firebase/firebase-tools/issues/3468

I, too, find this to be a fatal problem.

**Comment by yuchenshi** [comment]
Hi, thanks for filing this feature request! We are unable to promise any timeline for this, but if others also have this request, adding a +1 on this issue can help us prioritize adding this to the roadmap.

(Googler-only internal tracking bug: b/210056051)

**Comment by virtualmason** [comment]
+1

**Comment by camilaleniss** [comment]
+1 It will make the dev process easier instead of having to deploy it just for testing

**Comment by r-dev-limited** [comment]
Yeah pubsub emulator still does not support scheduled functions/retry and there is not even proper docs in emulator suite explaining setup and limitations :(

**Comment by kevpie** [comment]
+1 I also burned several hours due to this not being explicitly called out. Even after pouring over a lot of documentation and public posts.

**Comment by apetta** [comment]
+1

**Comment by ChrisGibb** [comment]
+1

**Comment by ykemer** [comment]
+1 Please

**Comment by ChrisGibb** [comment]
+1

**Comment by tomas-carv-com** [comment]
What is update on this @bkendall ? 

I mean we have now couple option to define retry mechanism, but it seems there is no way to run these in local environment

**Comment by lucasgm1997** [comment]
+1

**Comment by davidoort** [comment]
+1

**Comment by Fedorrychkov** [comment]
+1
